runtime/autoload/provider/clipboard.vim

```diff
@@ -5,42 +5,81 @@ let s:copy = {}
 let s:paste = {}
 
 function! s:try_cmd(cmd, ...)
-  let out = a:0 ? systemlist(a:cmd, a:1, 1) : systemlist(a:cmd, [''], 1)
+  let argv = split(a:cmd, " ")
+  let out = a:0 ? systemlist(argv, a:1, 1) : systemlist(argv, [''], 1)
   if v:shell_error
-    echo "clipboard: error: ".(len(out) ? out[0] : '')
-    return ''
+    echohl WarningMsg "clipboard: error: ".(len(out) ? out[0] : '')
+    return 0
```

Comment:
Because `"` starts a comment. I guess you need to ask whose idea it was to make `"` start a comment (or, more likely, whose idea it was to make `"` start a string in Vim: guess comment character comes from Vi which did not have +eval).